README.md

```diff
@@ -61,35 +63,51 @@ The get mode will show you the resolution of all active displays
     Available Modes on Display 1
     <snip>
 
-The set command takes a list of modes.  It will apply the modes
+The `set` command takes a list of modes.  It will apply the modes
 in the list of modes to the list of displays, starting with 0.
 Modes in excess of the number of active displays will be ignored.
 If you wish to set a monitor but not the lower numbered displays,
-there is a keyword 'skip' which can be subsituted for a resolution.
+there is a keyword `skip` which can be subsituted for a resolution.
 This keyword will cause the first display to be skipped.  If you
 specify more resolutions than you have active screens, the extra
 resolutions will be ignored.
 
-Example 1:
+
+## Examples
+
+__Example 1:__
+>
     This example works with one or more screens
     $ screenresolution set 800x600x32
-Result 1:
+
+>Result:
```

Comment:
how this formats is not ideal, imo.  The text result: should not be in the quote.  Otherwise this looks good and happy to merge!